
PACKAGE = "first_project"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("target_frame", str_t, 0, "Target frame for lidar data", "wheel_odom") 

edit_method=gen.enum([gen.const("wheel_odom", str_t, "wheel_odometry_frame", "Wheel odometry frame"),
                      gen.const("gps_odom", str_t, "gps_odometry_frame", "GPS odometry frame")],
                      "A frame selection enum")

exit(gen.generate(PACKAGE, "lidar_visualization_node", "lidar_visualization_node"))